tests/test_dict.py

```diff
@@ -37,6 +37,16 @@ def test_dict_unicode_key() -> None:
     assert ormsgpack.unpackb(ormsgpack.packb(obj)) == obj
 
 
+def test_dict_bytearray_value() -> None:
+    obj = {"key": bytearray(b"value")}
+    assert ormsgpack.unpackb(ormsgpack.packb(obj)) == obj
+
+
+def test_dict_memoryview_value() -> None:
+    obj = {"key": memoryview(b"value")}
+    assert ormsgpack.unpackb(ormsgpack.packb(obj)) == obj
+
+
```

Comment:
No need to add ad hoc tests. It is enough to add entries for `bytearray` and `memoryview` in `test_types.py.TYPE_PARAMS`.

Reply:
Ah didn't see that. I removed `bytearray` as it can't be used dict key anyways.